recipes/bdrc_crop_images.py

```diff
@@ -3,12 +3,13 @@
 import logging
 import boto3
 import prodigy
+from tools.config import s3_client1, PAGE_CROPPPING_BUCKET
```

Comment:
i would import from tools import config and then avoid writing line 10 and 11 on line 46 i would write config.s3_client1 and on 48 config.PAGE_CROPPING_BUCKET